lib/brainstem/api_docs/formatters/markdown/helper.rb

```diff
@@ -69,6 +69,15 @@ def md_li(text, indent_level = 0)
           def md_a(text, link)
             "[#{text}](#{link})"
           end
+
+
```

Comment:
Comment: It's so weird that the spacing is a double newline between each method.

Reply:
Just keeping the pattern consistent with Rob's work. Don't want to spend time removing double newlines everywhere.

Reply:
I think spending a little time on this file would be beneficial in the future. Keeping a maintained codebase will help future contributors understand desirable patterns. Although I understand that combing through the codebase is time consuming, we should tackle the problem piecewise so we can just "turn on" linting in the future without a hitch.